demo-app/src/app/shared/malihu-scrollbar/services/malihu-scrollbar.service.ts

```diff
@@ -0,0 +1,47 @@
+import { Injectable } from '@angular/core';
+
+type ScrollElement = string | JQuery | HTMLElement;
+
+@Injectable()
+export class MalihuScrollbarService {
+
+
+  constructor() { }
+
+  initScrollbar(element: ScrollElement, options: MCustomScrollbar.CustomScrollbarOptions) {
+    this.getElement(element).mCustomScrollbar(options);
+  }
+
+  scrollTo(element: ScrollElement, parameter: any, options: MCustomScrollbar.ScrollToParameterOptions) {
+    this.getElement(element).mCustomScrollbar('scrollTo', parameter, options);
+  }
+
+  update(element: ScrollElement) {
+    this.getElement(element).mCustomScrollbar('update');
+  }
+
+  stop(element: ScrollElement) {
+    this.getElement(element).mCustomScrollbar('stop');
+  }
+
+  disable(element: ScrollElement) {
+    this.getElement(element).mCustomScrollbar('disable');
+  }
+
+  destroy(element: ScrollElement) {
+    this.getElement(element).mCustomScrollbar('destroy');
+  }
+
+  private getElement(element: ScrollElement): JQuery {
```

Comment:
Not really sure it worth the pain and I would'nt know where to put it ... an helper module doesnt sound right?! If somebody needs it he can extract it